### PR TITLE
add some unit test cases

### PR DESCRIPTION
**Pull request by gy95**
Signed-off-by: gy95 <guoyao17@huawei.com>

**What type of PR is this?**
/kind test

**What this PR does / why we need it**:
add some unit test cases to improve the function coverage.
**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: cloud/pkg/edgecontroller/messagelayer/util_test.go [hunk 1]
```diff
@@ -69,6 +91,59 @@ func TestBuildResource(t *testing.T) {
...
+		resourceID   string
+	}
+	tests := []struct {
+		name         string
+		args         args
+		wantResource string
+		wantErr      error
+	}{
+		{
+			"BuildResourceForRouter(): Case 1: no resourceType.",
```
Comment (Iceber) [inline cloud/pkg/edgecontroller/messagelayer/util_test.go:106]: typo: "TestBuildResourceForRouter"

Comment (gy95) [reply cloud/pkg/edgecontroller/messagelayer/util_test.go]: ok. done.

**Review by fisherxu — COMMENTED** [review]
/approve

**Review by fisherxu — COMMENTED** [review]
/lgtm

**Pull request merged**